packages/lesswrong/server/gwwcFundraisers/cron.ts

```diff
@@ -0,0 +1,67 @@
+import ElectionCandidates from "../../lib/collections/electionCandidates/collection";
+import { addCronJob } from "../cronUtil";
+import { Globals } from "../vulcan-lib";
+
+type ElectionCandidateAmounts = {
+  amountRaised: number,
+  // TODO pull in the targetAmount here also
```

Comment:
Ticket for this followup: https://app.asana.com/0/628521446211730/1205898096818799/f